importers/yelp.ca/generate_csv.py

```diff
@@ -0,0 +1,127 @@
+import lxml.html as html
+import os
+import json
+import itertools
+from utils import get_text_from_css
+import re
+import csv
```

Comment:
While testing this, I came across a problem with some unicode encodings.

Instead of importing csv, import unicodecsv and it should fix the problem.
```
import unicodecsv as csv
```

Also, install unicodecsv if you don't already have it.  I did so by running python -m pip install unicodecsv